content/en/configuration/cloud2cloud-gateway.md

```diff
@@ -0,0 +1,280 @@
+---
+title: 'Cloud2Cloud Gateway'
+description: 'Service configuration overview'
+date: '2021-10-01'
+lastmod: '2021-10-01'
+categories: [configuration, deployment]
+keywords: [configuration]
+menu:
+  docs:
+    parent: configuration
+    weight: 70
+toc: true
+---
+
+C2C Gateway implements [Open Connectivity Foundation's Cloud API for Cloud Services](https://openconnectivity.org/developer/specifications/) providing API's to mirror user's devices to a different OCF standardized cloud.
```

Comment:
point `Open Connectivity Foundation's Cloud API for Cloud Services` to right doc

Reply:
Reason why I pointed to /specification is that your link points to v2.2.4. And we don't want to update documentation every time OCF releases a new spec.

Reply:
Then you can point to https://openconnectivity.org/specs/OCF_Cloud_API_For_Cloud_Services_Specification.pdf

Reply:
Done

Reply:
I dont see any update.